sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -608,6 +611,17 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .andWhere("application.applicationStatus != :overwrittenStatus", {
         overwrittenStatus: ApplicationStatus.Overwritten,
       });
+    // If institution id is present, get only the applications
+    // linked with the institution.
+
+    if (institutionId) {
```

Comment:
Suggestion, we can move the where(line 610,611) beyond line 624 to follow the pattern of select, join and where because sometimes where would have a dependency with join(not in this case)

Reply:
mm, I find this way more readable like till 610, its the default join, and where are beyond the logic and w.r.t conditions, so, keeping as it is

Reply:
As there is a technical reason behind my suggestion, I would still ask for the other way unless team thinks differently

@andrewsignori-aot @andrepestana-aot @guru-aot ?

Reply:
I would prefer to keep it as it is right now because IMO it makes it easier to read and identify the difference for the `if (institutionId)`.
Breaking the first part of the query in this scenario would add a bit more visual complexity.